src/execution/__tests__/errorPropagation-test.ts

```diff
@@ -0,0 +1,74 @@
+import { describe, it } from 'mocha';
+
+import { expectJSON } from '../../__testUtils__/expectJSON.js';
+
+import type { PromiseOrValue } from '../../jsutils/PromiseOrValue.js';
+
+import { parse } from '../../language/parser.js';
+
+import { buildSchema } from '../../utilities/buildASTSchema.js';
+
+import { execute } from '../execute.js';
+import type { ExecutionResult } from '../types.js';
```

Comment:
```suggestion
import { expectJSON } from '../../__testUtils__/expectJSON.js';
import type { PromiseOrValue } from '../../jsutils/PromiseOrValue.js';
import { parse } from '../../language/parser.js';
import { buildSchema } from '../../utilities/buildASTSchema.js';
import { execute } from '../execute.js';
import type { ExecutionResult } from '../types.js';
```

Reply:
[Looks like lint didn't like that](https://github.com/graphql/graphql-js/actions/runs/13496436014/job/37704680944?pr=4348)...

Reply:
Ugh 😅 we have really annoying lint rules and we should really get rid of some of them

Reply:
`pnpm lint --fix` should have you covered though

Reply:
Yes, if adds it back that's okay. Was looking at other test files and assumed the grouping was okay, I can take care of that later as well when I tweak the rules

Reply:
[Done](https://github.com/graphql/graphql-js/pull/4348/commits/aec37aa057e950871821be917a854eb1cacab639)